bin/workflows/pycbc_create_bank_verifier_workflow

```diff
@@ -182,18 +190,50 @@ inp_bank.description='TEMPLATEBANK'
 inp_bank.ifo_list=(['H1','L1','V1'])
 inp_bank.segment = workflow.analysis_time
 
-# Inspinj job
-inspinj_job = LalappsInspinjExecutable(workflow.cp, 'injection', out_dir='.',
-                                       ifos='HL', tags=[])
-
-def add_banksim_set(workflow, file_tag, num_injs, curr_tags, split_banks):
+# Injection Job
+# determine which executable will be used to generate injections
+injection_generator = workflow.cp.get('workflow', 'injection-generator')
+
+supported_injection_generators = ['inspinj', 'pycbc_create_injections']
+
+if injection_generator not in supported_injection_generators:
+    err_msg = """'injection-generator = {}' is not supported. \
+Please choose from = {}
+""".format(injection_generator,supported_injection_generators)
+    raise ValueError(err_msg)
+logging.info("Running with injection_generator = {}".format(injection_generator))
+
+if injection_generator == "inspinj":
+    sec_name = "inspinj_injection"
+    injection_job = LalappsInspinjExecutable(workflow.cp, sec_name, out_dir='.',
+                                        ifos='HL', tags=[])
+elif injection_generator == "pycbc_create_injections":
+    sec_name = "pycbc_injection"
+    injection_job = PycbcCreateInjectionsExecutable(
+        workflow.cp, sec_name, out_dir=".", ifo="HL", tags=[])
+
+def add_banksim_set(workflow, file_tag, num_injs, curr_tags, split_banks,
+                    injection_generator):
     """Add a group of jobs that does a complete banksim.
     """
-    inspinj_job.update_current_tags(curr_tags)
-    t_seg = segments.segment([1000000000, 1000000000+int(num_injs)])
-    node = inspinj_job.create_node(t_seg)
+    injection_job.update_current_tags(curr_tags)
+    start_time = 1000000000
+    end_time = 1000000000+int(num_injs)
+    t_seg = segments.segment([start_time, end_time])
+    if injection_generator == "inspinj":
+        node = injection_job.create_node(t_seg)
+        inj_file = node.output_file
+    elif injection_generator == "pycbc_create_injections":
+        # PycbcCreateInjectionsExecutable gets the segment length
+        # from the 'workflow' section of the ini file.
+        # Here we override the values from the ini file as they don't
+        # get used for anything else.
+        workflow.cp.set("workflow", "start-time", str(start_time))
+        workflow.cp.set("workflow", "end-time", str(end_time))
+        inj_file_ext = workflow.cp.get("pycbc_injection", "output-file-format")
```

Comment:
This option should go into `workflow_injection` in line with the config file guidelines in the PyCBC docs.
It should also go in the `PycbcCreateInjectionExecutable` class.